Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -420,24 +420,37 @@ void LoadEventNexus::exec() {
   }
 }
 
-std::pair<DateAndTime, DateAndTime>
-firstLastPulseTimes(::NeXus::File &file, Kernel::Logger &logger) {
+DateAndTime firstPulseTime(::NeXus::File &file, Kernel::Logger &logger) {
   file.openData("event_time_zero");
+  auto info = file.getInfo();
+  if (info.dims[0] == 0) {
+    std::string message =
+        "event_time_zero has zero dimension. Cannot establish run "
+        "start.";
+    logger.warning(message);
+    throw std::runtime_error(message);
+  }
 
-  if (!file.hasAttr("offset"))
-    throw std::runtime_error("No ISO8601 offset attribute provided");
+  DateAndTime offset;
+  std::string isooffset; // ISO8601 offset
 
-  const auto heldTimeZeroType = file.getInfo().type;
-  // Nexus only requires event_time_zero to be a NXNumber, we support two
-  // possibilities for held type
+  if (file.hasAttr("offset")) {
+    file.getAttr("offset", isooffset);
+    offset = DateAndTime(isooffset);
+  } else if (file.hasAttr("start")) {
+    file.getAttr("start", isooffset);
+    offset = DateAndTime(isooffset);
+  } else
+    offset = DateAndTime(DateAndTime::UNIX_EPOCH);
```

Comment:
Echoing my comment above. We threw before if we didn't know the offsets. Is it defined for ESS files that if offset is not present then it should be treated as Unix epoch?

Reply:
Yes ESS nexus files assume the unix epoch

Reply:
Ok. Thanks for clarifying.